Resources/Maps/_Sunrise/planet_prison.yml

```
@@ -15536,6 +15360,96 @@ entities:
     - type: Transform
       pos: 3.5,-7.5
       parent: 2
+  - uid: 2594
+    components:
+    - type: Transform
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)